install/kubernetes/helm/istio/values-istio-gateways.yaml

```diff
@@ -51,6 +51,8 @@ gateways:
       #requests:
       #  cpu: 1800m
       #  memory: 256Mi
+    cpu:
```

Comment:
Can you please add some comments here to clarify that cpu request needs to be set if want the HPA works, would be great to link Kubernetes HPA document here.

Reply:
It's done